src/data/selector.ts

```diff
@@ -91,8 +91,11 @@ export interface ColorRGBSelector {
 
 export interface ColorTempSelector {
   color_temp: {
-    min_mireds?: number;
-    max_mireds?: number;
+    unit: "Kelvin" | "Mired";
+    min?: number;
+    max?: number;
```

Comment:
Should be we explicit here? `min_kelvin`, `max_kelvin`?

Reply:
I so far decided against that just in case there might in the future be another third unit besides Mires and Kelvin, so we would not have to introduce additional min/max vars but just a new unit.

Reply:
I don't know, it feels weird when there is no unit set...

Reply:
Default unit would still be Mireds to keep backwards compatibility (although HA officially moved to Kelvin only since 2022.11).

But I can also rename to min_kelvin/max_kelvin, but then also setting a unit along side it as a separate parameter feels weird/redundant.

Reply:
> I so far decided against that just in case there might in the future be another third unit besides Mires and Kelvin, so we would not have to introduce additional min/max vars but just a new unit.

I think this is a good point.
Perhaps the code for setting the default min, max and step values also can be modified for this scenario.
Maybe a dictionary lookup with the unit as the key?
This reduces the need for (hard to read) ternary operator stuff.

Reply:
Also note that Mireds and Kevin are inversely proportional so a min_kelvin is a max_mired - could be a cause for confusion.